src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,46 @@
+@use "../../variables" as *;
+.container {
+  padding: 56px 0px 100px;
+  @media screen and (min-width: $breakpoint-lg) {
+    padding-bottom: 35px;
+  }
+}
+.message {
+  padding: 0px 20px 32px;
+  margin-bottom: 20px;
+  @media screen and (min-width: $breakpoint-lg) {
+    margin-bottom: 0px;
+  }
+  .title {
+    color: $orange_bistre;
+    margin-bottom: 0px;
+    padding-bottom: 6px;
+  }
+  .explain {
+    margin-bottom: 0px;
+  }
+}
+.wrapper {
+  @media screen and (min-width: $breakpoint-lg) {
+    display: flex;
+    gap: 20px;
+    margin-bottom: 40px;
+  }
```

```suggestion
.wrapper {
  gap: 24px;
  margin-bottom: 24px
  @media screen and (min-width: $breakpoint-lg) {
    display: flex;
    gap: 20px;
    margin-bottom: 40px;
  }
```
せっかく`wrapper`を作っていただいているので、Molecules/Planの下部についているであろう`margin-bottom: 20px`を取り除いて、lgのときと同様の記述にするのが、直感的にはわかりやすいかなと感じました。
ただ、見た目としては問題ないので次回以降意識してみてください！